src/client/java/hadoop-daos/src/test/resources/java-test.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/sh
+
+cd /root/daos
+first=$1
+shift
+caller=$1
+
+if [ "jvm" == "$caller" ]
+then
+        shift
+        out=$1
+        err=$2
+        shift 2
+        props=$*
+        echo $props
+
+        java -Xmx2048m -cp ./hadoop-daos-0.0.1-SNAPSHOT-shaded.jar:./hadoop-daos-0.0.1-SNAPSHOT-tests.jar $props  com.intel.daos.hadoop.fs.perf.Main $first 1>$out 2>$err
```

Comment:
(style)  line over 80 characters
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]